### ISSUE TITLE
Continue the project?

### ISSUE THREAD

**Issue by danielsanfr** [mayuso/Qomodoro#6]
Hi, how are you?
I would like to know if there is any interest in continuing this project?

I have some ideas and we could continue with your project.
I am also in contact with this other project to try to join forces: https://github.com/lmartinking/TomatoTimer/issues/1

**Comment by mayuso** [comment]
> I would like to know if there is any interest in continuing this project?

Yes. I revisited this project a couple days ago and I really want to continue developing it. The last months have been tough and although I had time, I didn't have the motivation needed to keep working on personal projects like this on top of my studies, full time job, and all the quarantine stuff going on. But things are getting easier and I want to start putting some time on open source projects again.

The first thing that I think needs some work is the User Interface, it is functional, does the trick, but it's the ugliest thing I've ever seen. So I'll be putting some work in my spare ours onto that in the next couple of days or (hopefully) weeks.

Any ideas and collaborations with other projects will be really welcome. I'd love to turn this into a feature-rich and robust project, it's something I currently use from time to time, but if it was better, I'd use it daily.

**Comment by danielsanfr** [comment]
I am happy to know that you are interested in continuing development.

> but it's the ugliest thing I've ever seen

I must admit that I thought that too 😂.

Let's wait a while to see if the person from the other project responds to my issue. Because I saw that that project is very advanced and I think we could make the two projects into one.

**Comment by mayuso** [comment]
Hey, 

I'm closing this issue for two reasons:

- The other project seems to be even less active than this one, so I don't expect any answer. I might take some ideas from them though, their project is really interesting and some of their features would work really well in this software too.

- I've already started working on reworking the UI, I am just a software engineer, so I have no idea how to make UI look and feel good, but I'm sure I can do better than what we had previously.

This is the old UI:

![qomodoro_old](https://user-images.githubusercontent.com/15814568/94477428-2b879d80-01d2-11eb-8f44-1a7f7836fe63.png)

This is the current **Work in progress**:

![qomodoro_wip](https://user-images.githubusercontent.com/15814568/94477494-465a1200-01d2-11eb-8378-777914742536.png)

I really won't invest *too much time* on this project because I already have other stuff that I am working on, but the UI really needed a rework, so it didn't feel right leaving it as ugly as it was. My short-term objective is to release a 0.2 version with improved visuals, something that I (or anyone) could use to track their time and maybe use it as a reference or small example of Qt development (I used it to learn anyway). 

**TL;DR**: Expect a release soon-ish (I hope) but not really huge new features. I am not really interested in continuing this project, but I want to leave it as a decent piece of software.
